src/main.rs

```diff
@@ -268,7 +266,19 @@ async fn run_upload(
     }
 
     Retry::spawn(default_delay(), || {
-        trunk_analytics_cli::clients::put_bundle_to_s3(&upload.url, &bundle_time_file)
+        put_bundle_to_s3(&upload.url, &bundle_time_file)
+    })
+    .await?;
+
+    let remote_urls = vec![repo.repo_url.clone()];
```

Comment:
Is this a correct order? Should we create repo first and then create upload?

Reply:
The order shouldn't matter since the other endpoints don't depend on a repo existing.